troi/playlist.py

```diff
@@ -1,79 +1,148 @@
 import ujson
 import openpost
+import requests
 
-from troi import Recording
+from troi import Recording, Playlist, PipelineError, Element
 from troi.operations import is_homogeneous
-import troi
 
+LISTENBRAINZ_PLAYLIST_CREATE_URL = "https://test.listenbrainz.org/1/playlist/create"
 
-class PlaylistElement(troi.Element):
+def _serialize_to_jspf(playlist, created_for=None):
```

Comment:
would be great to not duplicate this code between troi and LB. Any thoughts?

Reply:
Nothing easy, sadly.

Reply:
as we discussed, we should look at moving this to an external python package that both troi and LB can import